themes/bootstrap3/js/lightbox.js

```diff
@@ -474,7 +474,7 @@ VuFind.register('lightbox', function Lightbox() {
   // Element which to focus after modal is closed
   var _beforeOpenElement = null;
   function reset() {
-    _html(VuFind.loading());
+    _html('');
```

Comment:
No. I originally removed the default spinner to avoid double spinners when the lightbox is opened from a form submit button. Now that the ajax method adds the spinner in any case, it makes sense to let it handle it unconditionally. And making the modal empty makes the check for loading message style easier; if lightbox is empty, only a single line "Loading..." message is added since the overlay would look silly without underlying content.